app/models/content_store_organisation.rb

```diff
@@ -0,0 +1,61 @@
+require 'active_model'
+
+class ContentStoreOrganisation
+  include ActiveModel::Model
+
+  attr_reader :content_item
+
+  def initialize(content_item)
+    @content_item = content_item
+  end
+
+  def live?
+    organisation_state == 'live'
+  end
+
+  def self.find!(base_path)
+    content_item = ContentItem.find!(base_path)
+    new(content_item)
+  end
+
+  def organisations_count(department_type)
+    @content_item.content_item_data["details"][department_type].count
+  end
+
+  def organisations_list(department_type)
+    @content_item.content_item_data["details"][department_type]
+  end
+
+  def organisation_department_count(organisation)
+    organisation["works_with"].flatten.count
+  end
+
+  def get_name_of_organisation(organisation_id)
+    org_name = {
```

Comment:
We should move this declaration to a class level constant so that we don't instantiate a new hash each time we use this method.  We could think about a more descriptive name for it too.